cfn-templates/cid-cfn.yml

```diff
@@ -61,6 +61,8 @@ Metadata:
         default: "Deploy KPI Dashboard"
       OptimizationDataCollectionBucketPath:
         default: "Path to Optimization Data Collection S3 bucket"
+      MonthsLookback:
+        default: "Number of months to lookback in summary_view"
```

Comment:
Another topic is days to loopback in resource_view and hourly_view. Currently 30 but customers ask to control it as well

Reply:
I'll look into that as well. Thanks!